theme6/source/stylesheets/site.css.scss

```diff
@@ -0,0 +1,169 @@
+        border-radius: 3px;
+    }
+    #fb{
+      background: url("../images/fb.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-right: 23px;
+    }
+    #twt{
+        background: url("../images/twt.svg");
+        background-position: center center;
+        background-repeat: no-repeat;}
+    #g{
+        background: url("../images/g+.svg");
+        background-position: center center;
+        background-repeat: no-repeat;
+        margin-left: 23px;
+      }
+
+    .newsletter {
+      color: #72808C;
+      font-family: 'Montserrat', sans-serif;
+      font-size: 18px;
+      line-height: 21px;
+      text-align: center;
+    }
+
+    footer .box-social {
+      align-content: center;
+      width: 350px;
+    }
+
+    footer .box-social button:hover {
+      cursor: pointer;
+    }
+
+    footer .contenedorsocial {
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      width: 66%;
+    }
+
+    footer .contenedorsocial .logo {
+      margin-left: 96px;
+      margin-right: 96px;
+    }
+
+    footer .box-social p{
+      text-align: center;
+    }
+    footer .contenedornews{
+      width: 33%;
+      display: inline-flex;
+    }
+    footer .newsletter input {
+      height: 35px;
+      width:300px;
+      justify-content: center;
+    }
+
+    footer .information ul li a{
+      height: 22px;
+      width: 192px;
+      color: #72808C;
+      font-family:'Montserrat', sans-serif;
+      font-size: 18px;
+      line-height: 21px;
+      text-decoration: none;
+      align-items: center;
+      margin-right: 50;
+
+    }
+
+    footer .information ul li{
+      display: inline-block;
+    }
+    footer .information ul{
+      margin-top: 2%;
+      text-align: center;
+    }
+
+    footer .author {
+      display:block;
+      text-align: center;
+    }
+
+    footer .author p {
+      color: #72808C;
+      font-family:'Montserrat', sans-serif;
+      font-size: 16px;
+      line-height: 18px;
+    }
+/*=============Stylesheet MOBILE==========*/
+    @media screen and(max-width:320px) {
+      .information {
+        display: none;
+      }
+      .logo {
+        display: none;
+      }
+      #twt{
+        margin-left: -20px;
+        margin-right: -20px;}
+
+        .contenedorsocial {
+          display: flex;
+          margin-left: 50px;
+      }
+        .box-social {
+          align-content: center;
+        }
+        .box-social p {
+          font-size: 14px;
+        }
+        .boxsocialmedia {
+          width: 95%;
+          display: inline-flex;
+          justify-content: space-between;
+          align-content: center;
+        }
+
+        .contenedornews {
+          margin-top: 5px;
+          display: block;
+          align-content: center;
+          width: 100%;
+        }
+        .contenedornews p {
+          font-size: 15px;
+        }
+        .contenedornews input {
+          height: 40px;
```

Comment:
Properties should be ordered background-color, border, border-radius, height, width